nodejs/week1/meal_sharing/src/backend/routes/meals.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const router = express();
+
+const mealsJson = require(__dirname + '/../data/meals.json');
+const reviewsJson = require(__dirname + '/../data/reviews.json');
+
+mealsJson.forEach((meal) => {
+   meal.reviews = [];
```

Comment:
Please check this comment:

https://github.com/krithi-kondana/hyf-homework/pull/30#discussion_r382179162

Reply:
Thank you I was actually working on it today. I will integrate the changes you mentioned. I was wondering if there is a possibility that we can extract the functions with similar logics to into routers? For example throwing a random meal/app share the same function, can we use them as modules too?